lib/install/management/inspect.go

```diff
@@ -206,6 +206,7 @@ func (d *Dispatcher) GetDockerAPICommand(conf *config.VirtualContainerHostConfig
 					dEnv = append(dEnv, fmt.Sprintf("DOCKER_CERT_PATH=%s", abs))
 				}
 			} else {
+				log.Infof("")
```

Comment:
I would add a comment that this is purely for cosmetic purposes to improve logs readability. Otherwise it can be removed by another developer since it really looks redundant.